backend/src/seed/factories/donations.js

```diff
@@ -0,0 +1,18 @@
+import uuid from 'uuid/v4'
+
+export default function create() {
+  return {
+    factory: async ({ args, neodeInstance }) => {
+      const defaults = {
+        id: uuid(),
+        goal: 15000,
+        progress: 0,
+      }
+      args = {
+        ...defaults,
+        ...args,
+      }
+      return neodeInstance.create('Donations', args)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-12T15:39:40Z" title="Tuesday, November 12th 2019, 4:39:40 pm +01:00">Nov 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+import uuid from 'uuid/v4'
+
+export default function create() {
+  return {
+    factory: async ({ args, neodeInstance }) => {
+      const defaults = {
+        id: uuid(),
+        goal: 15000,
+        progress: 0,
+      }
+      args = {
+        ...defaults,
+        ...args,
+      }
+      return neodeInstance.create('Donations', args)
```


Usually I would prefer singular here. Like factory bot: https://github.com/thoughtbot/factory_bot The file is plural but the model is singular.

I think this model is an exception and I'm OK with it here.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-12T15:45:24Z" title="Tuesday, November 12th 2019, 4:45:24 pm +01:00">Nov 12, 2019</time>_

---


yeah, I thought of that as well, struggled with it, and just decided it could be `Donations` since it's not a single donation, but an aggregation of donations.